README.md

```diff
@@ -1,2 +1,4 @@
 # RouteSearchService
 A simple containerised micro service to search route between a given source and destination in a network of bus stations and routes.
+
+## No active code changes
```

Comment:
Why are we discontinuing this project? as per my understanding there are still active items on the road map.

Reply:
Oh okay, I didn't knew. I will revert my PR.